### PR TITLE
my Lesson 1 and 2 Task 2 done

### PR DESCRIPTION
**Pull request by iamipheco**
@promise-J
@ChidiPatrick
@OgJaykay
@Pieta9221
@MiniHack
@Melumm

I'm done with my Lesson 1 & 2 Task 2, Please Kindly Check it out.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+                    </nav>  
+                </article>
+                <img src="https://www.drweil.com/wp-content/uploads/2016/12/health-wellness_health-centers_men_can-laptops-harm-male-fertility_2714x1811_83681883.jpg" alt="a man with laptop">
+            </section>
+        </main>
+        <!-- The Form Section -->
+        <section>
+            <h2>If you need more articles like this , you can send us an email below</h2>
+            <form>
+                <label for="name">Name:</label>
```
Comment (Melumm) [inline docs/index.html:143]: Adding a placeholder attribute will make it easier for your client's user journey.

**Review by Melumm — COMMENTED** [review]
Good work! 🌿

File: docs/index.html (hunk 1 above)
Comment (iamipheco) [reply docs/index.html:143]: I didn't get you here, please can you kindly enunciate?

**Review by ChidiPatrick — APPROVED** [review]
GREAT! You can now merge

**Comment by iamipheco** [comment]
> GREAT! You can now merge

ALright Thanks

**Pull request merged**